### ISSUE TITLE
Pre-trained weights or deblurred results on the GoPro test set

### ISSUE THREAD

**Issue by pp00704831** [TencentCloud/ERDN#1]
Hello,

Congratulation on your work in ECCV 2022.
Do you plan to release the pre-trained weights or deblurred results on the GoPro test set?
